sdk/Sdk/ExtensionsCsprojGenerator.cs

```diff
@@ -49,14 +49,21 @@ private void WriteExtensionsCsProj(string filePath)
             File.WriteAllText(filePath, csprojContent);
         }
 
+
+
+
         internal string GetCsProjContent()
         {
             string extensionReferences = GetExtensionReferences();
 
+            string targetFrameWork = GetTargetFrameWork();
+
+            string netSdkVersion = GetNetSdkVersion();
+
             return $@"
 <Project Sdk=""Microsoft.NET.Sdk"">
     <PropertyGroup>
-        <TargetFramework>{_targetFramework}</TargetFramework>
+        <TargetFramework>{targetFrameWork}</TargetFramework>
```

Comment:
I think framework is one word so the original camelcase of the variable makes more sense, or is this something out of our control?